lib/archivist/vaults/mysql/index.js

```diff
@@ -88,7 +88,13 @@ MysqlVault.prototype.open = function (cb) {
 
 	this.pool = this.mysql.createPool(this.config);
 
-	setImmediate(cb);
+	this.pool.getConnection((err) => {
+		if (err) {
+			this.logger.emergency('Failed to open mysql connection', this.config);
```

Comment:
Any reason to log for MySQL but nothing else?

Reply:
Because the MySQL error was not explicit

> MySQL error

```
[MAGE] Error during MAGE setup: getaddrinfo ENOTFOUND myhost myhost:3306
```

> Couchbase error

```
[MAGE] Error during MAGE setup: The remote host refused the connection. Is the service up?
  error: {
    "code": "connectionRefused",
    "type": "CouchbaseError"
  }
```

Indeed, we could log an error for each vault.